packages/@ember/-internals/glimmer/lib/helpers/array.ts

```diff
@@ -0,0 +1,37 @@
+import { Arguments, VM } from '@glimmer/runtime';
+
+/**
+@module ember
+*/
+
+/**
+   Use the `{{array}}` helper to create an array to pass as an option to your
+   components.
+
+   ```handlebars
+   {{my-component people=(array
+     'Tom Dade'
+     'Yehuda Katz'
+     myOtherPerson)
+   }}
+   ```
+
+   Would result in an object such as:
+
+   ```js
+   ['Tom Date', 'Yehuda Katz', this.get('myOtherPerson')]
+   ```
+
+   Where the 3rd item in the array is bound to updates of the `myOtherPerson` property.
+
+   @method array
+   @for Ember.Templates.helpers
+   @param {Object} options
+   @return {Object} Array
+   @since 2.6.0
```

Comment:
Seems wrong, possibly 3.7?

Also can you add `@category` + your feature flag to the docs, that lets us filter out API docs for unlanded features...